src/signature_pad.js

```diff
@@ -451,16 +456,20 @@ SignaturePad.prototype._toSVG = function () {
           !isNaN(curve.control1.y) &&
           !isNaN(curve.control2.x) &&
           !isNaN(curve.control2.y)) {
+        const diff1x = curve.control1.x - curve.startPoint.x;
+        const diff1y = curve.control1.y - curve.startPoint.y;
+        const diff2x = curve.control2.x - curve.startPoint.x;
+        const diff2y = curve.control2.y - curve.startPoint.y;
+        const diffEx = curve.endPoint.x - curve.startPoint.x;
+        const diffEy = curve.endPoint.y - curve.startPoint.y;
+
         const attr = `M ${curve.startPoint.x.toFixed(3)},${curve.startPoint.y.toFixed(3)} `
-                   + `C ${curve.control1.x.toFixed(3)},${curve.control1.y.toFixed(3)} `
-                   + `${curve.control2.x.toFixed(3)},${curve.control2.y.toFixed(3)} `
-                   + `${curve.endPoint.x.toFixed(3)},${curve.endPoint.y.toFixed(3)}`;
+                   + `c ${diff1x.toFixed(3)},${diff1y.toFixed(3)} `
```

Comment:
likely lost in the shuffle is the subtle change from `C` (absolute coordinates) to `c` (relative coordinates)